docs/source/concept_models/vivarium_iv_iron/maternal_model/concept_model.rst

```diff
@@ -488,18 +495,26 @@ Where,
     - LCL is close to zero so it is possible to have negative draws assuming a normal distribution
     - Update to a truncated normal distribution instead (Ali to document and SWEs to implement)
     - Low priority since it hasn't broken anything yet (we're not selecting the negative draws)
-  * - Duration of postpartum period appears to be too long
-    - Unsure
-    - Investigate (both Ali and SWEs)
-    - Soon
   * - Pregnancy person time not stratified by pregnancy outcome
     - Not previously requested
-    - Ali to add to documentation and then SWEs to implement once it's added
+    - SWEs to implement (added to documentation of output table shell)
```

Comment:
We'll create a ticket for this.

Reply:
[MIC-2943](https://jira.ihme.washington.edu/browse/MIC-2943)

Reply:
Awesome, thank you!!